Description: Fix file pattern in pstoimg
 This patch fixes the file pattern in pstoimg to collect the files created by
 ghostscript.
Author: Holger Ruckdeschel <holger@hoicher.de>
Bug-Debian: http://bugs.debian.org/298329

--- latex2html-2008-debian1.orig/pstoimg.pin
+++ latex2html-2008-debian1/pstoimg.pin
@@ -1006,7 +1006,7 @@ sub pstoimg {
       print qq{$prompt: Error: Could not open directory "$pnmdir": $!\n};
       return 0;
       }
-    my @list = grep(/^\d+_\w*\./,readdir(DIR));
+    my @list = grep(/^\d+_${pnmbase}\.pnm/,readdir(DIR));
     closedir(DIR);
     if(@list) {
       my $i;

